lib/auth.coffee

```diff
@@ -29,6 +30,23 @@ getAuth = (deps, opts) ->
 	###
 	exports.twoFactor = twoFactor
 
+	userDetails = null
+	exports.getUserDetails = ->
```

Comment:
If this is supposed to be a user-accessible function, it should have docs like the others. If it's not, it shouldn't be exported.

Reply:
I guess that this needs to be exported either way, since it's used in other files.
If we document this, then we should probably reject with a `ResinNotLoggedIn` error, like the rest get<UserDetailProperty> methods do and in that case we should be able to remove the respective `throw`s from the rest of the methods.

Reply:
Ahh, I didn't notice it was used in other files.

That's actually easier though: it looks like it's only used in `application`, to get the user's id. If that is the reason, then imo really we should not export this at all; we already export `getUserId`, so we should just use that instead.